communication/mavlink_stream.h

```diff
@@ -90,9 +89,10 @@ typedef struct
 	byte_stream_t* rx;		///< Input stream
 	uint32_t sysid;			///< System ID
 	uint32_t compid;		///< System Component ID
-	mavlink_received_t rec;	///< Last received message
+	mavlink_received_t rec;		///< Last received message
 	bool msg_available;		///< Indicates if a new message is available and not handled yet
-	bool use_dma;			///< Indicates whether tx transfer should use dma
+	uint8_t mavlink_channel; 	///< Channel number used internally by mavlink to retrieve incomplete incoming message
+	bool debug;
```

Comment:
Fixed in https://github.com/lis-epfl/MAVRIC_Library/commit/86ef4fbee61beb7ef4be67c82ef8446c7481be16